.gitignore

```diff
@@ -14,4 +14,7 @@
 # Dependency directories (remove the comment below to include it)
 vendor/
 
-/*.pb.go
\ No newline at end of file
+/*.pb.go
+
+# IDE files
+.idea/
```

Comment:
Try keep generic/local ignore rules in a global ignore file https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Thanks for the tip!